themes/django20/layouts/index.html

```diff
@@ -11,26 +11,27 @@
   <section class="events" id="events">
     <div class="container container-events {{ .Params.cssClassSuffix }}">
       <article class="{{ .Params.cssClassSuffix }}">
-        
-        <h1>Venue events</h1>
-        
-        {{ $events := (where (.Site.GetPage "/events").Pages.ByDate "published" "!=" "false") }}
 
-        <table class="event-table">
+        {{ $events := .Site.Data.events }}
+
+        {{ partial "events_map.html" (dict "events" $events "context" .) }}
+
+        <table id="events" class="event-table">
           <thead>
           <tr>
-            <th>When</th>
-            <th>Where</th>
+            <th class="date">When</th>
             <th>What</th>
+            <th>Community</th>
           </tr>
           </thead>
           <tbody>
-            {{ range sort $events ".Params.event_date" "asc" }}
-              <tr>
-                <td>{{ time.Format "2006-01-01" .Params.event_date }}</td>
-                <td>{{ .Params.city }}</td>
-                <td><a href="{{ .Permalink }}" title="{{ .Title }}">{{ .Title }}</a></td>
-              <tr>
+            {{ range sort $events ".fields.date" "asc" }}
+              {{ $event := .fields }}
+                <tr>
+                  <td>{{ $event.date }}</td>
```

Comment:
I'd like to keep some sort of date formatting, so to revert to the previous `{{ time.Format "2006-01-01" .Params.event_date }}` - see: https://gohugo.io/functions/time/format/